tests/data_context/store/test_validation_config_store.py

```diff
@@ -50,10 +52,18 @@ def cloud_backed_store(cloud_details: CloudDetails):
 
 
 @pytest.fixture
-def validation_config() -> ValidationConfig:
+def validation_config(
+    in_memory_runtime_context: EphemeralDataContext,
+) -> ValidationConfig:
+    context = in_memory_runtime_context
+    batch_config = (
+        context.sources.add_pandas("my_datasource")
+        .add_csv_asset("my_asset", "data.csv")
+        .add_batch_config("my_batch_config")
+    )
     return ValidationConfig(
         name="my_validation",
-        data=BatchConfig(name="my_batch_config"),
+        data=batch_config,
```

Comment:
Need the parent/grandparent relationship now that serialization is happening properly. Using idiomatic instantiation pattern here